.github/workflows/create-quantic-package.yml

```diff
@@ -0,0 +1,54 @@
+name: Create Quantic Package Workflow
+
+env:
+  SFDX_AUTH_JWT_INSTANCE_URL: https://login.salesforce.com
+  SFDX_AUTH_JWT_USERNAME: sfdc.integration.devv2.hub@coveo.com
+  SFDX_AUTH_JWT_KEY: ${{ secrets.SFDX_AUTH_JWT_KEY_DEV }}
+  SFDX_AUTH_JWT_CLIENT_ID: ${{ secrets.SFDX_ATH_JWT_CLIENT_ID }}
+  PACKAGE_ID: 0Ho6g000000k9g8CAA
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '30 17 * * *'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  create-and-promote-quantic-package:
+    name: Create and Promote Quantic Package
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Check Out Repository
+        uses: actions/checkout@v2
+      - name: Install NPM dependencies
+        run: npm i -g sfdx-cli
+      - name: Create cache file
+        run: |
+          mkdir check-SHA
+          echo ${{ github.sha }} > github-sha.txt
+      - name: Check SHA
+        id: check_sha
+        uses: actions/cache@v1
+        with:
+          path: check-SHA
+          key: check-SHA-${{ github.sha }}
+      - name: Create jwt key file
+        if: steps.check_sha.outputs.cache-hit != 'true'
+        run: echo "$SFDX_AUTH_JWT_KEY" > server.key
+      - name: Connect to DevHub organization
+        if: steps.check_sha.outputs.cache-hit != 'true'
+        run: sfdx force:auth:jwt:grant --clientid $SFDX_AUTH_JWT_CLIENT_ID --jwtkeyfile server.key --username $SFDX_AUTH_JWT_USERNAME --setdefaultdevhubusername
+      - name: Create package version
+        if: steps.check_sha.outputs.cache-hit != 'true'
+        run: |
+          output=$(sfdx force:package:version:create -p $PACKAGE_ID -x)
```

Comment:
Do we really need the report? we can also just add a `-w [X minutes]` to the command to wait for it to finish?

Reply:
I think people usually also format the output as JSON in order to easily parse the result

Reply:
> Do we really need the report? we can also just add a `-w [X minutes]` to the command to wait for it to finish?

Wow yeah if that just sets a timeout and returns once create status is known then that's like... way better. 😅 my bad fred

Reply:
https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_force_package.htm

`[-w WAIT]`

Reply:
Yeah for some reason I didn't think that actually blocked the command from returning. I thought it just set a timeout to prevent the process from hanging ¯\_(ツ)_/¯

Reply:
Ohh yea well it will print the status of the creation every like 30s until it's completed and then the command returns